Jenkinsfile

```diff
@@ -38,6 +38,17 @@ pipeline {
             }
           } finally {
             sh "docker network rm ${networkName}"
+          }
+          docker.build('maven').inside() {
+                maven cmd: "-f deploy/single-project/pom.xml clean install " +
+                  "-Divy.engine.list.url=${params.engineListUrl} " +
+                  "-Divy.engine.version=[9.3.0,9.4.0\\) " 
+          }          
+          docker.build('maven').inside() {
+                maven cmd: "-f deploy/single-project-over-http/pom.xml clean install " +
+                  "--settings deploy/single-project-over-http/settings.xml " +
+                  "-Divy.engine.list.url=${params.engineListUrl} " +

```

Comment:
some duplication here: 44+49 ... you may easily share this in a variable:
```
def ivyParams= "-Divy.engine.list.url=${params.engineListUrl} " +
                  "-Divy.engine.version=[9.3.0,9.4.0\\) " 
docker.build('maven').inside() {
                maven cmd: "-f deploy/single-project/pom.xml clean install ${ivyParams}"
```

Reply:
I will do on the new PR.